---
layout: default
title: Joelle Mbatchou's Webpage
---

<section id="intro" class="main">
    <div class="content">
        <header class="major">
            <h2>Welcome!</h2>
        </header>
        <p>
        <span class="image right"><img src="images/Regeneron_logo.png" alt="" /></span>
I currently work as a statistical geneticist at the Regeneron Genetics Center where my research focuses on developing statistical methods and computational tools for large-scale genetic association analyses to better understand the impact of genetic variation on human disease. 
        </p> 
        <p>
        <span class="image right"><img src="images/uchicago.png" alt="" /></span>
        Prior to joining Regeneron, I obtained my PhD in Statistics from the University of Chicago under the supervision of Prof. Mary Sara McPeek where my research was centered on building statistical models for genetic association analysis in structured samples. 
        </p> 
        <p>
        You can read more about my recent work <a href="#first">below</a>, and
        if you want to get in touch with me, contact me by <a href="mailto:joelle.mbatchou@gmail.com">email</a>. <br>
        Thank you for visiting!</p>
        <!--
            <ul class="actions">
            <li><a href="generic.html" class="button">Learn More</a></li>
            </ul>
        -->
    </div>
</section>

<!-- First Section -->
<section id="first" class="main special">
    <header class="major">
        <h2>Research</h2>
        <p>
        My research is centered on genetic association analysis in structured samples, where a primary goal is to identify association between genetic markers (e.g. SNPs) and a phenotype of interest (e.g. complex human trait or gene expression levels). Sample structure can arise through the presence of indiviuals from different population subgroups (population stratification) or individuals who are related, and needs to be accounted for when performing association analyses to avoid getting inflated type 1 error. More recently, I have been interested in building efficient models and tools for application in large-scale biobanks (e.g. 100,000s of individuals and 1,000s of phenotypes).
        </p>
    </header>
    <ul class="features">
        <li>
            <span class="icon major style1 fa-folder-open"></span>
            <h3>Permutation Methods in Binary Trait Association Mapping</h3>
            <p>Identifying genetic associations involves assessing the statistical significance of a given test statistic by deriving its null distribution or an asymptotic approximation to it. However, this is not always feasible as the distribution may be intractable, which can arise for example with region-based tests (e.g. rare variant tests).
            A further limitation can arise when significance needs to be assessed for the maximum of many correlated tests, which is often encountered in genome scans to establish a genome-wide significance threshold.
            We develop a novel method to overcome these limitations when testing association between a binary trait and an arbitrary predictor in samples with population structure and/or related individuals.
            </p>
        </li>
        <li>
            <span class="icon major style3 fa-folder-open"></span>
            <h3>Fast Method for Assessing Significance for a Wide Class of Association Tests</h3>
            <p> We focus on assessment of significance in genetic association analysis of single or multi-dimensional phenotypes, including high-dimensional phenotypes, where we consider test statistics of a certain form, with association being tested with either a single genetic marker or with multiple genetic markers simultaneously. Existing approaches are either computationally burdensome (permutation-based approaches), or do not perform well in settings such as small samples, high-dimensional traits, or misspecified phenotype model, or do not account for existing sample structure when it is present. We develop a novel method to assess significance for a broad class of test statistics currently used in gentic association analysis that is based on moment-matching and allows for very general population structure and relatedness in the sample.

            </p>
        </li>
    </ul>

    <h2><b>Publications</b></h2>
    <div class="table-wrapper">
        <table>
            <tbody>
                <tr>
                    <td align="left"><b>J. Mbatchou</b>, M. Abney, and M. S. McPeek. <a href="https://doi.org/10.1101/451377" target="_blank">“Permutation methods for assessing significance in binary trait association mapping with structured samples.”</a> <i>bioRxiv</i> (2019): 451377.</td>
                </tr>
                <tr>
                    <td align="left">Jiang, D., <b>J. Mbatchou</b>, and M. S. McPeek. <a href="https://doi.org/10.1159/000446957" target="_blank">“Retrospective Association Analysis of Binary Traits: Overcoming Some Limitations of the Additive Polygenic Model.”</a> <i>Human heredity</i> 80, no. 4 (2016): 187-195.</td>
                </tr>
                <tr>
                    <td align="left">Nelsen, M. P., R. Lücking, <b>J. S. Mbatchou</b>, C. J. Andrew, A. A. Spielmann, and H. T. Lumbsch. <a href="https://doi.org/10.1007/s13225-011-0144-7" target="_blank">"New insights into relationships of lichen-forming Dothideomycetes."</a> <i>Fungal Diversity</i> 51, no. 1 (2011): 155-162.</td>
                </tr>
                <tr>
                    <td align="left">Kharas, G. B., E. Hanawa, B. L. Hill, K. T. Flynn, H. Husain, <b>J. Mbatchou</b>, W. Paik et al. <a href="https://doi.org/10.1080/10601325.2011.562456" target="_blank">"Novel Copolymers of 4-Fluorostyrene. 8. Some Ring-Trisubstituted 2-Phenyl-1, 1-dicyanoethylenes."</a> <i>Journal of Macromolecular Science</i>, Part A 48, no. 5 (2011): 327-331.</td>
                </tr>
                <tr>
                    <td align="left">Nelsen, M. P., R. Lücking, E. Rivas Plata, and <b>J. S. Mbatchou</b>. <a href="https://doi.org/10.1639/0007-2745-113.4.742" target="_blank">"Heiomasia, a new genus in the lichen-forming family Graphidaceae (Ascomycota: Lecanoromycetes: Ostropales) with disjunct distribution in Southeastern North America and Southeast Asia."</a> <i>The Bryologist</i> 113, no. 4 (2010): 742-751.</td>
                </tr>
                <tr>
                    <td align="left">Nelsen, M. P., R. Lücking, M. Grube, <b>J. S. Mbatchou</b>, L. Muggia, E. Rivas Plata, and H. T. Lumbsch. <a href="https://dx.doi.org/10.3114%2Fsim.2009.64.07" target="_blank">"Unravelling the phylogenetic relationships of lichenised fungi in Dothideomyceta."</a> <i>Studies in Mycology</i> 64 (2009): 135-144
                    </td>
                </tr>
                <tr>
                    <td align="left">Schoch, C. L., P. W. Crous, Johannes Z. Groenewald, E. W. A. Boehm, T. I. Burgess, J. De Gruyter, G. S. De Hoog et al. <a href="https://dx.doi.org/10.3114%2Fsim.2009.64.01" target="_blank">"A class-wide phylogenetic assessment of Dothideomycetes."</a> <i>Studies in mycology</i> 64 (2009): 1-15.</td>
                </tr>
            </tbody>
        </table>
    </div>
</section>

<!-- Second Section -->
<section id="second" class="main special">
    <header class="major">
        <h2>My resume</h2>
    </header>
    <p>For more details about my career, find my resume attached below.</p>

    <footer class="major">
        <ul class="actions">
            <li><a href="resume2019.pdf" class="button" target="_blank">Learn more</a></li>
        </ul>
    </footer>
</section>

<!-- Last section -->
<section id="blog" class="main special">
    <header class="major">
        <h2>My blog</h2>
        <p></p>
    </header>
    <footer class="major">
        <ul class="actions">
            <li><a href="#" class="button special">Coming soon!</a></li>
        </ul>
    </footer>
</section>


